[SCI-24554][NGL] Add API Call to turn Continuous Argon Purge (#18)
SciAps/SciApsApi: README.md

$ curl -X POST -H "Content-Type: application/json" -d @settings.json --output output.json http://192.168.42.129:8080/api/v2/acquire/all?mode=Alloy
```

#### /api/v2/purge/start

> This endpoint is used to start argon purge. This is used either to flush the argon line at power on,

Reviewer: ```suggestion
This endpoint is used to start argon purge, which can be used to flush the argon line 
```
